load-tests/README.adoc

```diff
@@ -0,0 +1,23 @@
+= Hawkular Metrics Load Tests
+:type: article
+
+A simple load testing tool for Hawkular Metrics
+
+== Why a Maven project?
+
+Because it makes it easier for developers to fire a load test:
+there's no need to download, extract and configure PATH.
+
+== The scenario
```

Comment:
The README should indicate that you need to start up Cassandra and Hawkular separately.